pipelines/_ingredients/ApPipe.yaml

```diff
@@ -80,6 +80,14 @@ subsets:
       - diffimTaskCore
     description: >
       An alias of ApPipe to use in higher-level pipelines.
+  promptQaMetrics:
+    subset:
+      - analyzeAssocDiaSrcCore
+      - analyzeTrailedDiaSrcCore
+      - diffimTaskCore
+    description: >
+      QA metrics to run in Prompt Processing.
+
```

Comment:
I don't think you need to redefine this; since you're not excluding any of these tasks on import, the definition from lsst/analysis_tools#218 should carry over.

Reply:
This addition was in response to your comment "A qaMetrics subset might be useful...". Did I misinterpret your request? You are correct that everything works without this extra subset definition.

Reply:
Sorry, when I wrote that I didn't realize you were planning to move (had already moved?) the definition to `analysis_tools`, as previously suggested.

TLDR: it might be useful to have a definition, but there's definitely no need to define it in more than one place.